src/views/GovernanceReviewTeam/Dates/index.tsx

```diff
@@ -53,29 +58,30 @@ const Dates = ({ systemIntake }: { systemIntake: SystemIntakeForm }) => {
     const newGrtDate = DateTime.fromObject({
       day: Number(grtDateDay),
       month: Number(grtDateMonth),
-      year: Number(grtDateYear)
+      year: Number(grtDateYear),
+      zone: 'UTC'
     });
+
     const newGrbDate = DateTime.fromObject({
       day: Number(grbDateDay),
       month: Number(grbDateMonth),
-      year: Number(grbDateYear)
+      year: Number(grbDateYear),
+      zone: 'UTC'
     });
 
-    const data = {
-      ...systemIntake,
-      grtDate: newGrtDate,
-      grbDate: newGrbDate,
-      id: systemId
-    };
-    dispatch(saveSystemIntake({ ...data }));
-
-    history.push(`/governance-review-team/${systemId}/intake-request`);
+    mutate({
+      variables: {
```

Comment:
ah yes, if you add the second type arg above, it will enforce the typing of this input object